### PR TITLE
some root docs improvements [MERGE_WHEN_READY]

### PR DESCRIPTION
**Pull request by dsvoronin**
remove unused :meta

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,14 +2,22 @@
 
 ## Disclaimer
 
-Используй на свой страх и риск:
+Right now this project is just a way for us to work as open as possible,\
+and share ideas and its implementation with community.
```
Comment (eugene-krivobokov) [inline README.md:6]: ideas and their implementation with the community.
?

**Review by eugene-krivobokov — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (dsvoronin) [reply README.md:6]: не, вроде its (ideas же)

Comment (eugene-krivobokov) [reply README.md:6]: singular: its, plural: theirs

Comment (dsvoronin) [reply README.md:6]: slojna

**Comment by eugene-krivobokov** [comment]
closes #75

**Pull request merged**